distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -2709,7 +2709,7 @@ For example, installing the `security-sts-ldaplogin` feature adds an ldap realm.
 |Activates public key infrastructure authentication
 
 |anon(anonymous)
-|provides anonymous access
+|provides anonymous/guest access
```

Comment:
I think we should just say guest in the documentation instead of anonymous/guest.